### DISCUSSION TITLE
缓存过期时间是指的什么缓存

### DISCUSSION THREAD

**Discussion by dary123** [AlistGo/alist#4127] [General]
缓存过期时间是指的什么缓存，是alist账号token过期时间吗

**Comment by anwen-anyi** [comment]
你指的的那个缓存过期时间，是添加驱动时候的吗？那个是刷新网盘缓存的 你从网盘对网盘内容进行了变更，但是AList不会立刻更新，会在距离时间到了才会刷新，如果你是着急在AList使用网盘变更后的内容 你可以在前端页面进入你需要的网盘里面右下角有个刷新按钮 这样会立即刷新 可以看到你网盘变更后的内容

如果你需要alist账号token过期时间 那你需要在 `config.json` 配置文件设置 里面默认是48小时 你可以改成你需要的 单位是小时

**Reply by gaijiaqi** [reply]
解决大问题了 找了半天不知道怎么实时获取刚离线的文件,挂在到本地的盘怎么才能刷新呢?

**Reply by qian9008** [reply]
网盘开启了本地代理，是缓存了被点击的文件和目录结构？对于本地webdav缓存设置为0 就是和smb效果一样是吧。。
